packages/docusaurus-theme-classic/src/theme/DocSidebar/Desktop/index.tsx

```diff
@@ -24,16 +24,19 @@ function DocSidebarDesktop({path, sidebar, onCollapse, isHidden}: Props) {
   } = useThemeConfig();
 
   return (
-    <div
+    <nav
       className={clsx(
         styles.sidebar,
         hideOnScroll && styles.sidebarWithHideableNavbar,
         isHidden && styles.sidebarHidden,
-      )}>
-      {hideOnScroll && <Logo tabIndex={-1} className={styles.sidebarLogo} />}
+      )}
+      aria-label="docs sidebar">
+      {hideOnScroll && (
+        <Logo tabIndex={-1} className={styles.sidebarLogo} aria-hidden="true" />
```

Comment:
From my understanding, the `aria-hidden="true"` shouldn't always be applied.




When the site navbar is shown, then that logo is behind the navbar so it's hidden:

![CleanShot 2025-03-28 at 10 16 14](https://github.com/user-attachments/assets/ce173f5b-b393-411c-a9ba-3393f585dd90)

However when the site gets scrolled and the navbar disappears, this logo becomes visible, and is an actual link to the site homepage. In this case I don't believe applying `aria-hidden` is a good idea

![CleanShot 2025-03-28 at 10 16 50](https://github.com/user-attachments/assets/ae5b0e85-4422-4bd1-8771-c5ab55d5bd34)